go/vt/vtgate/semantics/vtable.go

```diff
@@ -37,18 +37,14 @@ var _ TableInfo = (*vTableInfo)(nil)
 // dependencies implements the TableInfo interface
 func (v *vTableInfo) dependencies(colName string, org originable) (dependencies, error) {
```

Comment:
nit: the function does not return error from any of the implementations, hence the signature can be changed.